.buildbot.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+#
+# Build script for continuous integration.
+
+set -e
+mkdir out && cd out
+cmake -GNinja -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS=OFF -Denable_parallel_mark=ON -Denable_gc_assertions=ON ../
+ninja
```

Comment:
Can we run the test suite in buildbot?

Reply:
Good idea. Fixed here: 0ef204dd0b1bc782650dd4213012a3eac43561bb